spec/routing/bulletins_routing_spec.rb

```diff
@@ -20,7 +18,7 @@
 
     it 'routes to bulletins#sunday' do
       expect(show_sunday_bulletin).
-          to route_to(controller: 'basechurch/v1/bulletins',
+          to route_to(controller: 'v1/bulletins',
```

Comment:
Use 2 (not 4) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.